src/box/lua/call.c

```diff
@@ -1068,9 +1076,13 @@ lbox_func_new_or_delete(struct trigger *trigger, void *event)
 	return 0;
 }
 
-static struct trigger on_alter_func_in_lua = {
-	RLIST_LINK_INITIALIZER, lbox_func_new_or_delete, NULL, NULL
-};
```

Comment:
This trigger seems to be moved accidentally. Please, keep it in place.

Reply:
Fixed.